constellations/orion/src/repos/account_repo.go

```diff
@@ -117,32 +118,82 @@ func (acc *accountRepo) SelectAllNegativeBalances() ([]domains.AccountSum, error
 	return results, nil
 }
 
-func (acc *accountRepo) Insert(account domains.Account) error {
+func (acc *accountRepo) InsertWithUser(account domains.Account, user domains.User) error {
+	tx, err := acc.db.Begin()
+	if err != nil {
+		return err
+	}
 	utils.LogWithContext("accountRepo.Insert", logger.Fields{"account": account})
```

Comment:
Make sure to log first thing in every function. We want to call this log thing first, then start the transaction